README.md

```diff
@@ -6,3 +6,9 @@ XDF is a general-purpose container format for multi-channel time series data wit
 XDF is open source and designed to be a community project; that is, everyone is invited to contribute to the format [specification](https://github.com/sccn/xdf/wiki/Specifications). The core specification of the format is kept as simple as possible, but at the same time rich enough to support features such as time stamp corrections, boundary chunks for seeking, and data streams supporting many different data types (including int, float, double, and string).
 
 The XDF concept and beta version was developed by Christian Kothe and Clemens Brunner at the [Swartz Center for Computational Neuroscience (SCCN)](http://sccn.ucsd.edu/) at the [University of California San Diego, CA, USA](http://www.ucsd.edu/).
+
+# Repository Structure
+
+The XDF specification is outlined in the home repository at https://github.com/sccn/xdf . As of late april 2019, XDF readers, writers, and associated tools can be found in the https://github.com/xdf-modules organization.
```

Comment:
The XDF specification is outlined in the home repository at https://github.com/sccn/xdf. As of late April 2019, XDF readers, writers, and associated tools can be found in the https://github.com/xdf-modules organization.

Reply:
What's the difference? Just the missing space? I do that intentionally when a URL is at the end of a sentence to simplify copy-pasting. I don't know what the use case is here, but it's a habit I've developed because copying URLs with touchscreens is painful, especially in iOS. If this is breaking some README parser then I'll change it, otherwise I'd rather leave it.

Reply:
Yes, the space and April with capitalized A. I've tried the link on iOS, it works perfectly without the space (long press on the link and a context menu will pop up). So yes, please change it.